Nodejs/Product/Nodejs/Project/NodejsProjectNode.cs

```diff
@@ -577,9 +577,9 @@ public NodejsProjectNode(NodejsProjectPackage package)
                 return false;
             }
 
-            var relativeFile = CommonUtils.GetRelativeFilePath(this.FullPathToChildren, fileNode.Url);
+            var relativeFile = CommonUtils.GetRelativeFilePath(this.FullPathToChildren, fileNode.Url);
             if (this._analyzer != null && this._analyzer.Project != null 
-                && this._analyzer.Project.Limits.IsPathExceedNestingLimit(relativeFile)) {
+                && this._analyzer.Project.Limits.IsPathExceedNestingLimit(relativeFile)) {
```

Comment:
can you verify that the line endings didn't change in this file? (I suspect that's why they're still showing up in the diff)